### DISCUSSION TITLE
How to increase log size?

### DISCUSSION THREAD

**Discussion by shalak** [hassio-addons/app-nut#379] [Q&A] [answered]
It looks like my addon logs are limited to 101 lines Even when I ssh to the "real" HassOS and do `docker logs addon_a0d7b954_nut` I only see 101 lines of the logs.

How do I increase this?

**Comment by frenck** [comment, marked as answer]
> How do I increase this?

You can't. This is not an add-on limition or question, but an Home Assistant one.

../Frenck

**Reply by shalak** [reply]
Do you know a way how to increase it, on Home Assistant side?

**Reply by frenck** [reply]
As the first sentence of my previous response already answered: you can't.
